### PR TITLE
I added hover and transition effects to the memory-card image container…

### PR DESCRIPTION
**Pull request by JosephOkpala**
…, kindly merge my branch. Thank you

### CODE REVIEW

File: gameBase/src/css/styles.css [hunk 1]
```diff
@@ -30,6 +30,14 @@ body {
   transition: transform 0.5s;
 }
 
+.memory-card img:hover {
```
Comment (GeraAlcantara) [inline gameBase/src/css/styles.css:33]: nice effect just can you adapted so only when the card is not flip have that hover effect, hint on the class flip remove the hover effect

**Review by GeraAlcantara — CHANGES_REQUESTED** [review]

**Comment by JosephOkpala** [comment]
Alright, I'm on it !

**Review by GeraAlcantara — APPROVED** [review]

**Comment by JosephOkpala** [comment]
I'm available for further tasks!

**Comment by JosephOkpala** [comment]
Thank You.

**Pull request merged**